projects/ng-tailwind/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "ng-tailwind",
-    "version": "1.2.246",
+    "version": "1.2.247",
```

Comment:
A alteração de versão, fazer somente quando já for publicar. Caso contrário, pode ser que alguém queira publicar uma versão e dê conflito com essa.

Reply:
Além disso, separar a alteração de versão em outro commit, por exemplo: "Update version -- 1.2.247". Dessa forma, fica mais fácil debugar os commits quando acontece algum problema em uma versão.